tools/check/ut.go

```diff
@@ -0,0 +1,534 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path"
+	"runtime"
+	"strings"
+	"sync"
+	"time"
+
+	// Set the correct when it runs inside docker.
+	_ "go.uber.org/automaxprocs"
+)
+
+func usage() {
+	msg := `// run all tests
+ut
+
+// show usage
+ut -h
+
+// list all packages
+ut list
+
+// list test cases of a single package
+ut list $package
+
+// run all tests
+ut run
+
+// run test all cases of a single package
+ut run $package
+
+// run test cases of a single package
+ut run $package $test
+
+// build all test package
+ut build
+
+// build a test package
+ut build xxx`
+	fmt.Println(msg)
```

Comment:
How about printing them to `stderr`?

Reply:
It's not handled in this PR...
Things to be done in the following PRs including:

> polish of the tool ut, exit code, output log etc